RIOT-OS/RIOT
# tests/sys/progress_bar: move configuration to app.config

### Contribution description

Providing the configuration via Makefile variables if Kconfig is supported by the module may involve some non-determinism in the environment variable that is finally exposed to the python tests. This commit moves the configuration from Makefile to app.config.

### Testing procedure

Confirm that the configuration from `app.config` is applied locally with `make -C tests/sys/progress_bar all test`

Merge and wait for the next nightly with fingers crossed.

### Issues/PRs references

Follow-up of #20763 and will need to follow-up to remove the debug prints in the python script.
